MSAL/src/native_auth/public/state_machine/error/MSALNativeAuthError.swift

```diff
@@ -26,13 +26,13 @@ import Foundation
 
 @objc
 public class MSALNativeAuthError: NSObject, LocalizedError {
+    public let identifier: String
```

Comment:
Why not expose this to Objective-C? I understand as you mention in the PR description that you can only use it in a `switch case` in Swift but it might still be useful in Objective-C code.

Reply:
I see all of you folks agree on this, I will expose it.